.github/workflows/coverage.yml

```diff
@@ -1,9 +1,11 @@
-name: Coverage check on main push
+name: Coverage Check
 
 on: [push]
 
 env:
     COVERAGE_SENSITIVITY_PERCENT: 1
+    MAINNET_RPC: ${{ secrets.MAINNET_RPC }}
+    SEPOLIA_RPC: ${{ secrets.SEPOLIA_RPC }}
```

Comment:
why do we need RPCs for coverage? shouldn't we make it 100% unit tests?

Reply:
true